src/app/sign-up/sign-up.page.html

```diff
@@ -52,7 +52,7 @@
               <ion-label>
                 Phone Number
               </ion-label>
-              <ion-input type="text" placeholder=" Enter Phone Number" name="phonenumber" [(ngModel)]="phonenumber" required>
+              <ion-input type="text" placeholder=" Enter Phone Number (e.g. +12223334444)" name="phonenumber" [(ngModel)]="phonenumber" required>
```

Comment:
may need to change the placeholder to only +12223334444 given the width of the screen we have